waldo/upload.go

```diff
@@ -42,33 +49,24 @@ func NewUploadAction(options *UploadOptions, ioStreams *lib.IOStreams) *UploadAc
 //-----------------------------------------------------------------------------
 
 func (ua *UploadAction) Perform() error {
-	ciMode := ua.detectCIMode()
-	args := os.Args[1:]
+	if ua.options.LegacyVersion {
```

Comment:
Speaking of the legacy options, what is LegacyHelp doing ? Is it used anywhere ?

Reply:
It is automatically handled by the Cobra library.